ecs-cli/modules/cli/compose/entity/service/service_test.go

```diff
@@ -778,7 +1018,7 @@ func getCreateServiceWithDelayMockClient(t *testing.T,
 		DesiredCount:   aws.Int64(1),
 		RunningCount:   aws.Int64(1),
 		ServiceName:    aws.String("test-created"),
-		Deployments:    []*ecs.Deployment{
+		Deployments: []*ecs.Deployment{
```

Comment:
I indent it but when I save the file, it automatically reverts. Any idea what is going on?

Reply:
lol it's done by your IDE. Don't force it and let it be.

Reply:
I just have no idea what is causing it. Nothing much I can do about it lol

Reply:
more specifically it's probably GoFmt deciding something arcane about the indentation of struct literal members.

Reply:
Or if it is not consistent to the other ones. You can save it using other text editors.